pkg/detectors/azure_entra/common.go

```diff
@@ -0,0 +1,127 @@
+package azure_entra
+
+import (
```

Comment:
nit: Can we please sort these.

```
import (
	"fmt"
	"io"
	"net/http"
	"strings"

	regexp "github.com/wasilibs/go-re2"
	"golang.org/x/sync/singleflight"

	"github.com/trufflesecurity/trufflehog/v3/pkg/cache/simple"
	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
)
```

Reply:
I wonder if my Goland settings changed. I don't recall the import sorting+grouping being this atrocious.